language/English/strings.po

```diff
@@ -8838,7 +8850,43 @@ msgctxt "#19289"
 msgid "Hide group"
 msgstr ""
 
-#empty strings from id 19290 to 19498
+#. Used on recordings context menu
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19290"
+msgid "Undelete"
+msgstr ""
+
+#. Used on recordings context menu
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19291"
+msgid "Delete permanently"
+msgstr ""
+
+#. Used on recordings context menu
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19292"
+msgid "Delete all permanently"
+msgstr ""
+
+#. Used on yes no dialog
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19293"
+msgid "Delete all recordings permanently?"
+msgstr ""
+
+#. Used on yes no dialog header
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19294"
+msgid "Confirm undelete"
```

Comment:
from a usability POV, do we need to confirm an undelete action? I mean this action does no harm at all and I don't know of any OS that asks for confirmation to undelete something.